assets/js/components/Work/Tabs/Preservation.jsx

```diff
@@ -75,6 +77,20 @@ const WorkTabsPreservation = ({ work }) => {
   return (
     <div data-testid="preservation-tab">
       <UITabsStickyHeader title="Preservation and Access Masters" />
+      <DisplayAuthorized action="edit">
+        <div className="buttons is-right">
+          <Button
+            data-testid="button-new-file-set"
+            isPrimary
+            onClick={() => setIsModalHidden(!isModalHidden)}
+          >
+            <span className="icon">
+              <FontAwesomeIcon icon="file-image" />
+            </span>
+            <span>Add a FileSet</span>
```

Comment:
On button labels, for consistency (and this isn't like documented anywhere), but we've been using a convention to only have the first word capitalized.  One day we should clean it up, but it's probably 50/50 on whether "file set" is one word, camelCased, all that fun stuff.